dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -47,75 +38,83 @@ internal class RumSessionScope(
     private val sessionMaxDurationNanos: Long = DEFAULT_SESSION_MAX_DURATION_NS
 ) : RumScope {
 
-    internal val childrenScopes = mutableListOf<RumScope>()
-
-    internal var keepSession: Boolean = false
     internal var sessionId = RumContext.NULL_UUID
-    internal val sessionStartNs = AtomicLong(System.nanoTime())
-    internal val lastUserInteractionNs = AtomicLong(0L)
-
+    internal var sessionState: State = State.NOT_TRACKED
+    private val sessionStartNs = AtomicLong(System.nanoTime())
+    private val lastUserInteractionNs = AtomicLong(0L)
     private var resetSessionTime: Long? = null
 
     internal var applicationDisplayed: Boolean = false
```

Comment:
seems these 2 are not used anymore in this class

Reply:
Indeed